CMakeLists.txt

```diff
@@ -144,6 +144,10 @@ function(buildSdk)
         tar -czf ${PROJECT_SOURCE_DIR}/${PROJECT_NAME}_${SDK_VER}_${REV_HASH}_${PRISM_PLATFORM}.${CONNECT_BUILD_TYPE}.tar.gz ${DELIVERY_FILES}
         WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX})
 
+    add_custom_target(artifactory
+        jfrog rt u ${PROJECT_SOURCE_DIR}/${PROJECT_NAME}_${SDK_VER}_${REV_HASH}_${PRISM_PLATFORM}.${CONNECT_BUILD_TYPE}.tar.gz libs-release-local/${PRISM_PLATFORM}/
```

Comment:
You should use a variable to store the archive file name to avoid a discrepancy in naming for delivery targets.